src/Model/EventStream/EventStreamFilter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+namespace Neos\EventStore\Model\EventStream;
+
+use Neos\EventStore\Model\Event\EventTypes;
+use Webmozart\Assert\Assert;
+
+/**
+ * TODO
```

Comment:
Just the comment missing or something else?

Reply:
Not that I know of.
But I'm not so sure about the `for*` constructor.. Maybe we want to use the `create` and `with` pattern instead, that we use in the Subgraph filters?

Reply:
...however, if we keep the filter argument optional, the following two would be the same, which feels suboptimal to me:
`$eventStore->load($streamName)`
and
`$eventStore->load($streamName, EventStreamFilter::create())`

Reply:
discussed with @nezaniel  - we both think it is jnot a problem that the two calls above are effectively the same. Because if you create EventStreamFilter programmatically, the empty case must also be creatable and usable.

I'll take care of adjusting it towards this direction :)